lib/view/home_page.dart

```diff
@@ -108,66 +108,78 @@ class _HomePageState extends State<HomePage> {
-    final appBarBottom = appBarHeight.toDouble() - logoHeight.toDouble();
-    final scrollPos = _scrollPosition.round();
-    final percentage = scrollPos < appBarBottom
-        ? ((scrollPos) / appBarBottom.toDouble())
-        : 1.00;
+    final Widget mainLogo = SvgPicture.asset(
+      'assets/images/main_logo.svg',
+      height: logoHeight,
+    );
+    const appBarBottom = appBarHeight - logoHeight;
+    // On iOS scroll position can be negative due to over scroll
+    // So prevent negative scrollPos which results in a negative percentage
+    final scrollPos = _scrollPosition.isNegative ? 0.0 : _scrollPosition;
+    final percentage =
+        scrollPos < appBarBottom ? scrollPos / appBarBottom : 1.0;
 
-    return Builder(builder: (context) {
-      final _scrollController = PrimaryScrollController.of(context);
+    return Builder(
+      builder: (context) {
+        final _scrollController = PrimaryScrollController.of(context);
 
-      _scrollController.addListener(() {
-        setState(() {
-          _scrollPosition = _scrollController.position.pixels;
+        _scrollController.addListener(() {
+          setState(() {
+            _scrollPosition = _scrollController.position.pixels;
+          });
         });
-      });
 
-      return CustomScrollView(
-        slivers: <Widget>[
-          SliverAppBar(
-            // Warning brightness interacts with SystemUiOverlayStyle
-            // See system_bars.dart comments
-            brightness: Brightness.light,
-            automaticallyImplyLeading: false,
-            expandedHeight: appBarHeight.toDouble(),
-            bottom: PreferredSize(
-              preferredSize: Size.fromHeight(appBarHeight.toDouble()),
-              child: Padding(
-                padding: const EdgeInsets.fromLTRB(16, 0, 16, 4),
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: <Widget>[
-                    mainLogo,
-                    GestureDetector(
-                      onTap: () => InfoView.navigateTo(context),
-                      child: Icon(
-                        Icons.info_outline,
-                        size: 24,
-                        color: generateIconColor(AppColors.homeAppBarIcon,
-                            AppColors.appBarIcon, percentage),
-                      ),
-                    )
-                  ],
+        return CustomScrollView(
+          slivers: <Widget>[
+            SliverAppBar(
+              // Warning brightness interacts with SystemUiOverlayStyle
+              // See system_bars.dart comments
+              brightness: Brightness.light,
+              primary: true,
+              automaticallyImplyLeading: false,
+              expandedHeight: appBarHeight,
+              centerTitle: false,
+              forceElevated: false,
+              elevation: 0.0,
```

Comment:
This means that at least on android, we loose the shadow behind the app bar:

![image](https://user-images.githubusercontent.com/902252/79630684-b9c51580-8196-11ea-9ce9-595723b17a43.png)

However this does seem to be an acceptable tradeoff for solving the problem with black shadow on android.

You might want to have a look at this PR https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/pull/239
for a slightly better solution - at least with this one we have the elevation once we have scrolled along

Reply:
Actually … I think there may be been some problems with the PR I’ve linked, in that scrolling fast seemed to stuff it up.  If you do go down this route be sure to give it some good testing, so make sure my approach doesn't just stuff things back up

Reply:
Yep I changes elevation to 0.0 as it is also 0.0 on the nav bar. But I can put it back if you prefer.

Reply:
I can remove elevation easily and put it back. Doing it now.

The fix wasn't actually anything to do with elevation.

Reply:
forcedElevation doesn't actually do anything in this context so I'll remove it as well.

Reply:
OK so still some things to fix later when elevation is present. But for now eleavtion is 0.0.